subprojects/core/src/main/java/org/gradle/configuration/DefaultBuildConfigurer.java

```diff
@@ -19,11 +19,8 @@
 import org.gradle.api.internal.GradleInternal;
 import org.gradle.execution.ProjectConfigurer;
 import org.gradle.util.SingleMessageLogger;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class DefaultBuildConfigurer implements BuildConfigurer {
-    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultBuildConfigurer.class);
     private final ProjectConfigurer projectConfigurer;
 
     public DefaultBuildConfigurer(ProjectConfigurer projectConfigurer) {
```

Comment:
`@SuppressWarning` here.

Reply:
SuppressWarnings is only needed in projects using strict compilation (e.g. the tooling API). You don't need to check for this in a review, the compiler does it already.

Reply:
Good to know... so does this mean that we don't care about the deprecation warning outputted in gradle compile tasks?

Reply:
I think we haven't cared much so far, but I might be mistaken. Maybe we should just standardize on strict compilation. That would certainly make the logs shorter by forcing people to think about every warning.